### DISCUSSION TITLE
Feature Request: Version API Update

### DISCUSSION THREAD

**Discussion by shigma** [howmanybots/onebot#136] [想法 / 建议]
这个 API 与 QQ 本身关系不大，而是 CoolQ 插件相关，我觉得有必要在统一的协议中调整这个接口。由于这个调整是一个 breaking change，我会尽量避开大家已经使用的字段名。在当前大版本中，各应用可以在原有基础上增加这些字段作为返回值；而在下一个大版本中，则将这些字段作为正式的标准。

## get_version_info

| 字段名 | 数据类型 | 说明 |
| ----- | ------- | --- |
| `app_name` | string | 应用标识，如 `mirai-native` |
| `app_directory` | string | 应用根目录路径 |
| `app_version` | string | 应用版本号 |
| `cqhttp_version` | string | CQHTTP 版本，如 `4.15.0` |
| …… | …… | 可由其他应用扩展的字段 |

**Comment by stdrc** [comment]
ok
